bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/messages.properties

```diff
@@ -550,6 +550,8 @@ IDEWorkspacePreference_workspaceName=Wor&kspace name:
 IDEWorkbenchPreference_workbenchSystemExplorer=Command for launching system e&xplorer:
 IDEWorkspacePreference_UnknownNatureSeverity=Report unknown project n&ature as:
 IDEWorkspacePreference_UnknownNatureSeverity_Ignore=Ignore
+IDEWorkspacePreference_UnknownEncodingSeverity=Report missing project &encoding as:
```

Comment:
1. "Show workspace nam&e"
2. "New text line delimiter" -> "D&efault"

Options uses &e in the same dialog, I'm not sure how this collision is meant to be resolved (Alt+E results in the "Show workspace name" checkbox being interacted with). @iloveeclipse ? If there is already 1 collision that cannot be worked around, no point in adding another one.

Reply:
True.

I admittedly didn't test the mnemonic at all (apparently, there is [no mnemonic support in SWT for macOS](https://bugs.eclipse.org/bugs/show_bug.cgi?id=399554)).

Do we want to choose some other char? If so: which one is best? Or is it feasible to not specify one at all?

Reply:
What about the c?